### DISCUSSION TITLE
能不能出一个ios版的啊

### DISCUSSION THREAD

**Discussion by ChronosLynn** [MaaAssistantArknights/MaaAssistantArknights#10468] [Q&A] [answered]
找了好久 就是没看见ipa的啊啊啊啊啊啊啊啊

**Comment by BoredTape** [comment]
啊？真有这种需求吗？首先，maa是通过[Android Debug Bridge (adb)](https://developer.android.com/tools/adb)连接安卓设备（安卓模拟器），然后截图、识别、操作等等，如果说你是要用ios作为maa端那么目前阶段必须要解决这些问题为前提：

- 在ios下有adb相关工具包
- 一个可编译成ios版的相关maa项目

这2个问题目前解决的成本相对比较大，短期内应该不会有

在解决完上面的问题后。
如果你的需求是：ios设备既是作为maa端，也要同时运行游戏，由于ios的墓碑机制，可能会导致maa无法保持在后台运行。同时，设备分辨率的适配也是成本较大的。

以上仅仅是我个人观点。

**Comment by ABA2396** [comment, marked as answer]
你既然知道 ipa ，我猜你想找的是 https://github.com/MaaAssistantArknights/MaaMacGui ，下载和其他的一样，在主仓库里下 MAA-vx.y.z-macos-universal.dmg

文档： https://maa.plus/docs/zh-cn/manual/device/macos.html
